### PR TITLE
Remove the SPEC edit hack, replace with proper repos

### PR DESCRIPTION
**Pull request by Wuerfelbecher**
Replaced the `sed` specfile edit with the proper repos.

### CODE REVIEW

**Comment by joshuaboniface** [comment]
Yay :tada: @brianjmurrell this look good to you too?

File: deployment/centos-package-x64/docker-build.sh [hunk 1]
```diff
@@ -8,17 +8,9 @@ set -o xtrace
...
+# Build RPM
 make -f .copr/Makefile srpm outdir=/root/rpmbuild/SRPMS
-
-# Remove dep for nodejs/yarn since our build env won't have these (NVM instead)
-sed -i '/BuildRequires:  nodejs >= 8 yarn/d' SPECS/jellyfin.spec
-
-# Build the RPMs
-rpmbuild -bs SPECS/jellyfin.spec --define "_sourcedir ${SOURCE_DIR}/SOURCES/pkg-src/"
-rpmbuild -bb SPECS/jellyfin.spec --define "_sourcedir ${SOURCE_DIR}/SOURCES/pkg-src/"
+rpmbuild --recompile /root/rpmbuild/SRPMS/jellyfin-*.src.rpm
```
Comment (brianjmurrell) [inline deployment/centos-package-x64/docker-build.sh]: I generally use `rpmbuild --rebuild -bb <src_rpm>` here.  I've never used `--recompile`.  Is it producing an actual binary RPM?  The [docs](http://ftp.rpm.org/max-rpm/s1-rpm-b-command-other-build-commands.html) suggest to me that it doesn't.

Comment (Wuerfelbecher) [reply deployment/centos-package-x64/docker-build.sh]: Yes it seems to build a actual binary RPM, i intended to use the same `rpmbuild -rb` as with Fedora but the CentOS version didn't have it so used the first best from the `--help` text.

**Comment by Wuerfelbecher** [comment]
Should i also include the changes to build this on `master`?

File: deployment/centos-package-x64/docker-build.sh (hunk 1 above)
Comment (Wuerfelbecher) [reply deployment/centos-package-x64/docker-build.sh]: I replaced the flag with your's since it actually builds a binary RPM.

**Comment by joshuaboniface** [comment]
Thanks guys!

**Pull request merged**